go-app-ussd_registration.js

```diff
@@ -1658,7 +1681,7 @@ go.app = (function() {
         var data = {
         on_whatsapp: self.im.user.get_answer("on_whatsapp") ? "true" : "false",
         consent: self.im.user.get_answer("state_message_consent") === "yes" ? "true" : "false",
-        language: "en",
+        language: self.im.user.get_answer("state_language"),
```

Comment:
This means that we'll be changing from 2 character language codes like `en` to 3 character codes like `eng`. Just checking that the RapidPro flow that gets triggered from this can handle that.

Reply:
Yes. I had already made changes to the triggered flow to expect this.